src/components/Match/TeamTable.jsx

```diff
@@ -19,22 +37,31 @@ const TeamTable = ({
   direTeam = {},
   summable = false,
   hoverRowColumn = false,
-}) => (
-  <div>
-    <Heading
-      title={`${getTeamName(radiantTeam, true)} - ${heading}`}
-      icon={<IconRadiant />}
-    />
-    <Table data={filterMatchPlayers(players, 'radiant')} columns={columns} summable={summable} hoverRowColumn={hoverRowColumn} />
-    {picksBans && <PicksBans data={picksBans.filter(pb => pb.team === 0)} /> /* team 0 - radiant */}
-    <Heading
-      title={`${getTeamName(direTeam, false)} - ${heading}`}
-      icon={<IconDire />}
-    />
-    <Table data={filterMatchPlayers(players, 'dire')} columns={columns} summable={summable} hoverRowColumn={hoverRowColumn} />
-    {picksBans && <PicksBans data={picksBans.filter(pb => pb.team === 1)} /> /* team 1 - dire */}
-  </div>
-);
+  loggedInId,
+}) => {
+  const user = players.find(player => player.account_id === loggedInId);
+
+  return (
+    <StyledDiv user={user}>
+      <Heading
+        title={`${getTeamName(radiantTeam, true)} - ${heading}`}
+        icon={<IconRadiant />}
+      />
+      <span id="radiant">
+        <Table data={filterMatchPlayers(players, 'radiant')} columns={columns} summable={summable} hoverRowColumn={hoverRowColumn} />
```

Comment:
since this only affects the table, i'd prefer the table handle the styling/logic rather than wrapping this whole thing in the styled div.  You can pass down the ID to the table, or a function that lets it determine which row should be highlighted.